tests/snappi/pfcwd/files/pfcwd_multi_node_helper.py

```diff
@@ -136,7 +140,8 @@ def run_pfcwd_multi_node_test(api,
                      data_pkt_size=DATA_PKT_SIZE,
                      trigger_pfcwd=trigger_pfcwd,
                      pause_port_id=port_id,
-                     tolerance=TOLERANCE_THRESHOLD)
+                     tolerance=TOLERANCE_THRESHOLD,
+                     is_mlnx_4600c=is_mlnx_4600c)
```

Comment:
It seems to me that we should pass 'platform' rather than 'is_mlnx_4600c' as an argument to verify_results.